### PR TITLE
Adjust new categories for Italy

### PR DESCRIPTION
**Pull request by fansanelli**
please review

### CODE REVIEW

File: categories.json [hunk 1]
```diff
@@ -457,75 +457,75 @@
...
+          "areas": [ "DE", "AT", "CH", "FR" ],
           "form_details": [ "delivery", "access" ]
         },
         "child_clothes": {
           "name": "Child clothes",
           "osm_tags": [
             { "shop": "clothes", "clothes": "babies|children" }
           ],
-          "areas": "all",
+          "areas": [ "IT" ],
```
Comment (PanierAvide) [inline categories.json:484]: Might be `-areas` instead ?

Comment (fansanelli) [reply categories.json:484]: It's intended... All shop closed in Italy but babies. You have all clothing shops open.

Comment (PanierAvide) [reply categories.json:484]: If baby clothes shops are open in Italy, then you can let `"areas": "all"`, this means to display this in all countries including Italy. `"areas": [ "IT" ]` means display __only__ in Italy ;)

Comment (fansanelli) [reply categories.json:484]: I know this. But you don't needed it in France.. It is a subset. Anyhow should i change it or can you edit it?

Comment (PanierAvide) [reply categories.json:484]: OK ! I just understood, sorry for that. So that's good this way

**Pull request merged**